quadriga/src/main/java/edu/asu/diging/quadriga/service/impl/CollectionManager.java

```diff
@@ -0,0 +1,35 @@
+package edu.asu.diging.quadriga.service.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.quadriga.core.mongo.CollectionRepository;
+import edu.asu.diging.quadriga.domain.elements.Collection;
+import edu.asu.diging.quadriga.service.ICollectionManager;
+import edu.asu.diging.quadriga.web.forms.CollectionForm;
+
+@Service
+public class CollectionManager implements ICollectionManager {
+
+    @Autowired
+    private CollectionRepository collectionRepo;
+
+    
+    /**
+     * Creates a new Collection instance and stores it in mongodb
+     * 
+     * @param collectionForm   Form data that needs to be added to database
+     * 
+     * 
+     * @return Collection Instance that is saved in database
+     * 
+     **/
+    public Collection addCollection(CollectionForm collectionForm) {
+        Collection collection = new Collection();
+        collection.setName(collectionForm.getName());
+        collection.setDescription(collectionForm.getDescription());
+        collectionRepo.save(collection);
```

Comment:
you want to return the collection that the save method returns (I believe it is not void, right?) in case the caller needs the id of the newly created object (which is not guaranteed to be set in the passed object).

Reply:
not addressed